falcon/errors.py

```diff
@@ -2472,9 +2472,7 @@ class MediaValidationError(HTTPBadRequest):
     attribute using the "raise ... from" form when raising.
 
     Note:
-        All the arguments should be passed as keyword only. Using them as positional
-        arguments will raise a deprecation warning and will result in an
-        error in a future version of falcon.
+        All the arguments must be passed as keyword only.
```

Comment:
not sure if we need this note. probably just for the 3.x series, since it behaves differently from the others.
then on v4 we can cleanup or remove

Reply:
Yeah, I thought it wouldn't hurt to stress the difference. Yes, I was thinking we could bulk-remove all these notes when we drop non-kw-args in 4.0.